apps/aechannel/src/aesc_offchain_state.erl

```diff
@@ -229,13 +233,13 @@ run_extra_checks(F, Mod, Tx) when is_function(F, 2) ->
 
 -spec add_signed_tx(aetx_sign:signed_tx(), state(), aec_trees:trees(),
                     aetx_env:env(), map()) -> state().
-add_signed_tx(SignedTx, #state{signed_txs=Txs0}=State, OnChainTrees,
+add_signed_tx(SignedTx, #state{}=State, OnChainTrees,
```

Comment:
It's not `add_signed_tx` anymore, but rather `set_signed_tx` now, but I guess it's easier for backwards compatibility 🙂

Reply:
addressed